### ISSUE TITLE
className prop not listed in documentation

### ISSUE THREAD

**Issue by seanlindo** [jsdir/react-ladda#42]
Can this be added? I can make a PR.

**Comment by jsdir** [comment]
Good idea. In the prop description, be sure to mention that the `className` is automatically appended to `ladda-button` in order to retain the existing Ladda styles. We should also note that all props on `LaddaButton` (with [a few exceptions](https://github.com/jsdir/react-ladda/blob/710181e856c31fca8001140cfd79193f2dd7ce42/src/LaddaButton.jsx#L8)) are passed down to the internal `button` component automatically.

**Comment by seanlindo** [comment]
Will do, I'll work on this later in the week. 

Thanks!

**Comment by code-by** [comment]
Is this done?

**Comment by jsdir** [comment]
@code-by this was never finished
